apps/ledger-live-desktop/tests/page/account.page.ts

```diff
@@ -95,5 +98,19 @@ export class AccountPage extends AppPage {
     await this.scrollToOperations();
     await expect(this.lastOperation).toBeVisible();
     await expect(this.operationList).not.toBeEmpty();
+    if (await this.showMoreButton.isVisible()) await this.showMoreButton.click();
+  }
+
+  @step("Expect `derivate address index to be 0` to be visible")
+  async expectAddressIndexToBe0() {
```

Comment:
0 could be a parameter I assume
method could be named `"verifyAccountAddressIndex(int: number)"`

Reply:
yes 👍